core/src/main/java/com/scalar/db/common/AbstractDistributedTransactionManager.java

```diff
@@ -66,9 +72,17 @@ public Optional<String> getTable() {
     return tableName;
   }
 
-  protected DistributedTransaction activate(DistributedTransaction transaction)
+  protected DistributedTransaction decorate(DistributedTransaction transaction)
```

Comment:
`decorate()` method decorates the transaction object with the `DistributedTransactionDecorator` instances.